homework/index.js

```diff
@@ -1,19 +1,28 @@
 'use strict';
 
 {
-  function fetchJSON(url, cb) {
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', url);
-    xhr.responseType = 'json';
-    xhr.onload = () => {
-      if (xhr.status >= 200 && xhr.status <= 299) {
-        cb(null, xhr.response);
-      } else {
-        cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
-      }
-    };
-    xhr.onerror = () => cb(new Error('Network request failed'));
-    xhr.send();
+  async function fetchJSON(url) {
+    const response = await fetch(url);
+    if (!response.ok) {
+      throw new Error(
+        `Network error: ${response.status} - ${response.statusText}`,
+      );
+    }
+    return response.json();
+    // return new Promise((resolve, reject) => {
```

Comment:
"Dead code" should never be left in a PR. In real life, a PR is a request to pull your changes into the production master branch. Nobody is interested in what you tried and abandoned. You can always look at previous versions of your code through your Git commits.

Reply:
I am sorry @remarcmij . Next time I take account. Thank you.